tools/msp_merge/msp_merge.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="msp_merge" name="Merge MSP Spectra" version="0.1.0" python_template_version="3.5">
+   <creator>
+        <person
+            givenName="Wudmir"
+            familyName="Rojas"
+            url="https://github.com/wverastegui"
+            identifier="0000-0001-7036-9987" />
+        <person
+            givenName="Helge"
+            familyName="Hecht"
+            url="https://github.com/hechth"
+            identifier="0000-0001-6744-996X" />
+        <organization
+            url="https://www.recetox.muni.cz/"
+            email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+            name="RECETOX MUNI"/>
+    </creator>
+    <description>Merge MSP Spectra files</description>
+    <requirements>
+         <requirement type="package" version="0.14.0">matchms</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+       #set newinput = str("' '").join([str($f) for $f in $input])
+        python3  $__tool_directory__/msp_merge.py
+        --filenames    '$newinput'
+        --outfilename  '$outfile'
+    ]]></command>
+    <inputs>
+        <param name="filen" type="data" format="msp" multiple="true"/>
+    </inputs>
+    <outputs>
+        <data name="outfile" format="msp" />
+    </outputs>
+    <tests>
+         <test>
+            <param name="nfile" value="1-NITROPYRENE.msp,23-DICHLOROPHENOL.msp,24-DICHLOROPHENOL.msp" ftype="msp"/>
```

Comment:
looking at inputs `nfile` looks like a typo

Reply:
Second test (collection) does not pass

Reply:
Also fixed in https://github.com/RECETOX/galaxytools/pull/263/commits/b7645dedc493c20f05ae92a170d0ff47e922e81c